frontend/src/components/Workspace/Visualize/Plot/ImagePlot.tsx

```diff
@@ -554,36 +700,69 @@ const ImagePlotChart = memo(function ImagePlotChart({
   }
 
   const renderActionRoi = () => {
-    if (!roiDataState?.length || outputKey !== "cell_roi") return null
-    if (!isAddRoi) {
-      return <LinkDiv onClick={addRoi}>Add ROI</LinkDiv>
+    if (!roiFilePath || !roiFilePath.includes(CELL_ROI)) return null
+    if (action) {
+      return (
+        <>
+          {action !== "Add ROI" ? (
+            <BoxDiv>
+              ROI Selecteds: [{pointClick.map((item) => item.z).join(",")}]
+            </BoxDiv>
+          ) : null}
+          <BoxDiv sx={{ display: "flex", gap: 1 }}>
+            <LinkDiv
+              sx={{
+                color:
+                  action === "Delete ROI"
+                    ? "#F84E1B"
+                    : action === "Merge ROI"
+                    ? "#6619A9"
+                    : "default",
+                display: "flex",
+                gap: 1,
+                textDecoration: "none",
+                cursor: "default",
+              }}
+            >
+              {`[${action}]`}
+            </LinkDiv>
+            <LinkDiv onClick={onCancelAdd}>Cancel</LinkDiv>
+            <LinkDiv
+              style={{
+                opacity:
+                  (pointClick.length < 2 && action === "Merge ROI") ||
+                  (pointClick.length < 1 && action === "Delete ROI")
+                    ? 0.5
+                    : 1,
+                cursor:
+                  (pointClick.length < 2 && action === "Merge ROI") ||
+                  (pointClick.length < 1 && action === "Delete ROI")
+                    ? "default"
+                    : "pointer",
+              }}
+              onClick={addOrSelectRoi}
+            >
+              OK
```

Comment:
@sanglevinh 
After selecting OK in Merge ROI, Delete ROI, it seems that the screen does not transition to the TOP screen of Edit.  -> Please make it transition.
*I think it used to be transitioned before.

Reply:
@itutu-tienday  i fixed it, pls check it again